dvc/render/match.py

```diff
@@ -94,13 +108,24 @@ def match_defs_renderers(
 
             converter = _get_converter(renderer_cls, inner_id, props, definitions_data)
 
-            dps, rev_props = converter.flat_datapoints(rev)
+            try:
+                dps, rev_props = converter.flat_datapoints(rev)
+            except Exception as e:  # noqa: BLE001, pylint: disable=broad-except
```

Comment:
I am trying to match what we do on `error_handler`. I see `flat_datapoints()` reraising `DvcException` on some instances.

The question is, is there a potential to raise other errors like `OSError`, etc? And looking at the `image.flat_datapoints()`, it might raise other exceptions. So I don't want this to fail in any case.